services/movie.py

```diff
@@ -0,0 +1,46 @@
+from db.models import Movie
```

Comment:
Add blank line between different import groups
Correct ordering of imports according to pep8